From 3936b2a7e5dbf4235a5fdb9e82e75beb21d8e713 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Thu, 9 Apr 2015 17:05:24 +0800
Subject: [PATCH 0955/1594] MLK-10629-6 dts: imx7d-sdb: do not enable wlreg_on
 always

commit 9107da01bf3709fc3ab7a0c7f6864c429604e394 from
git://git.freescale.com/imx/linux-2.6-imx.git

Broadcom WiFi requires reset by operating wlreg_on regulator
when doing insmod/rmmod.
Keep wlreg_on will cause card state wrong and unable to re-insmod.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 6b466e4..261da36 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -80,8 +80,6 @@
 			regulator-name = "wlreg_on";
 			gpio = <&gpio4 21 GPIO_ACTIVE_HIGH>;
 			enable-active-high;
-			regulator-boot-on;
-			regulator-always-on;
 		};
 	};
 
-- 
1.7.5.4

